docs/data/migration/migration-tree-view-v6/migration-tree-view-v6.md

```diff
@@ -387,3 +387,14 @@ This will help create a new headless version of the `TreeItem` component based o
    )
  }
 ```
+
+### ✅ Rename `nodeId` to `itemId`
```

Comment:
Probably worth mentioning the `nodeId` prop received by the `ContentComponent` prop of `TreeItem` as well (it's in this PR from what I saw)